package.json

```diff
@@ -141,12 +141,16 @@
       "lcov"
     ],
     "collectCoverageFrom": [
-      "src/js/components/**/*.{js}",
-      "src/js/contexts/**/*.{js}",
-      "src/js/themes/**/*.{js}",
-      "src/js/utils/**/*.{js}",
-      "!src/js/components/**/*.{stories.js}",
-      "!src/js/components/**/stories/*.js"
+      "src/js/components/**/*.js",
+      "src/js/contexts/**/*.js",
+      "src/js/themes/**/*.js",
+      "src/js/utils/**/*.js",
+      "!src/js/**/**/*.stories.js",
+      "!src/js/**/**/stories/*.js",
+      "!src/js/**/**/stories/**/*.js",
```

Comment:
Isn't line #150 already covers line #149?

Reply:
The code coverage does change slightly. Here is the coverage numbers before:
78.48% Statements
69.56% Branches
74.63% Functions
79.61% Lines

Here are the coverage numbers after:
78.47% Statements
69.6% Branches
75.34% Functions
79.68% Lines

As for lines #150 and #151 they are covered by #149, I didn't catch that before. I will go back and remove them

Reply:
Actually I was wrong, I just tested it and line #151 is still needed. Without it `components/Select/stories/components` and `components/Select/stories/components/icons` are still included in the coverage report

Reply:
>The code coverage does change slightly. Here is the coverage numbers before:
78.48% Statements
69.56% Branches
74.63% Functions
79.61% Lines
>Here are the coverage numbers after:
78.47% Statements
69.6% Branches
75.34% Functions
79.68% Lines

Let's add this information to the PR description. it will be good practice for future PRs as well.

Reply:
Ok I updated the PR description